app/models/host_report.rb

```diff
@@ -45,8 +44,20 @@ class HostReport < ApplicationRecord
 
   default_scope -> { order('reported_at DESC') }
 
+  # TODO: temporary until we decide what will status bitfiled be exactly
+  def status
+    @status ||= case format
+                when 'puppet'
+                  JSON.parse(body)['metrics']['resources']['values']
```

Comment:
I had to add `rescue 0` in the end of this line and for ansible as well. Proabably some incorrect data in the DB?

Reply:
Yes, it will happen if you have Puppet report with a body not in `JSON.parse` compatible way stored.